src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.cs

```diff
@@ -58,15 +58,20 @@ public override async Task<Solution> GetModifiedSolutionAsync(Document document,
                 return document.Project.Solution;
             }
 
-            var suggestedFileNames = GetSuggestedFileNames(
-                state.TypeNode,
-                IsNestedType(state.TypeNode),
-                state.TypeName,
-                state.SemanticDocument.Document.Name,
-                state.SemanticDocument.SemanticModel,
-                cancellationToken);
-
-            var editor = Editor.GetEditor(operationKind, (TService)this, state, suggestedFileNames.FirstOrDefault(), cancellationToken);
+            var suggestedFileName = operationKind switch
+            {
+                MoveTypeOperationKind.RenameType => Path.GetFileNameWithoutExtension(document.Name),
+                MoveTypeOperationKind.MoveTypeNamespaceScope => document.Name,
```

Comment:
also don't understand this.  if we're moving the type... why do we just get he doc name, and why aren't ee trimming the extension like above?  this needs commenting.